### ISSUE TITLE
getLocalBindings doesn't always give types to all bindings

### ISSUE THREAD

**Issue by isovector** [haskell/haskell-language-server#1607]
Given

```haskell
codeActionProvider :: PluginMethodHandler IdeState TextDocumentCodeAction
codeActionProvider state plId (CodeActionParams _ _ (TextDocumentIdentifier uri) range _ctx)
  = _
```

the `IdentifierDetails` used in `getLocalBindings` for `state` and `plId` have types, but the `uri`, `range` and `_ctx` don't.

@wz1000 suggests this might be fixed in 8.10

**Comment by isovector** [comment]
This turns into a Wingman problem, where several things that are in scope don't end up in the `Hypothesis`.

**Comment by isovector** [comment]
This doesn't appear to be fixed in 8.10.4
